File: ELSE/SS/day10/line.cpp
#include <bits/stdc++.h>
using namespace std;
map<string, int> datas;
char str[20];
int n, m, nn, val;
int main()
{
    scanf("%d %d", &n, &m);
    for (int i = 0; i < n; i++)
    {
        scanf("%s %d", str, &val);
        if (val < m)
            val = m;
        datas[str] += val;
    }
    scanf("%d", &nn);
    for (int i = 0; i < nn; i++)
    {
        scanf("%s", str);
        if (datas.find(str) == datas.end())
        {
            printf("No Info\n");
        }
        else
        {
            printf("%d\n", datas[str]);
        }
    }
}

File: ELSE/SS/day10/line.py
datas={}
n,m=map(int,input().split())
for i in range(n):
    s,num=input().split()
    num=int(num)
    if num<m:
        num=m
    if s not in datas.keys():
        datas[s]=num
    else:
        datas[s]+=num
nn=int(input())
for i in range(nn):
    s=input()
    try:
        print(datas[s])
    except:
        print("No Info")